packages/tocco-ui/src/Panel/Panel.js

```diff
@@ -0,0 +1,57 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+/**
+ * <Panel/> is used to conceal and display related content alternating by interaction.
+ */
+class Panel extends React.Component {
+  state = {
+    isOpen: this.props.isOpen
+  }
+
+  toggleOpenState = () => {
+    // eslint-disable-next-line
+    this.props.isToggleable
+    && this.setState(prevState => ({
+      isOpen: !prevState.isOpen
+    }))
+  }
+
+  render() {
+    const {
+      isToggleable
+    } = this.props
+
+    return (
+      <div>
+        { // eslint-disable-next-line
+          React.Children.map(this.props.children, child => {
```

Comment:
Why do you disable eslint?

Is it because of missing propType for children? Please add them instead.

Please never user eslint-disable unless you are really sure.

Reply:
Yes and ok.

Reply:
see 88e5dbe45b2124693323e37e12e97f459bcf44ec